hazelcast/src/test/java/com/hazelcast/quorum/executor/ExecutorQuorumWriteTest.java

```diff
@@ -453,19 +455,14 @@ public void invokeAll_noQuorum() throws Exception {
     }
 
     @Test
-    public void invokeAll_timeout_quorum() throws Exception {
-        try {
-            expectQuorumException(exec(0).invokeAll(Arrays.<Callable<Object>>asList(callable(), callable()), 10l, TimeUnit.SECONDS));
-        } catch (UnsupportedOperationException ex) {
```

Comment:
no reason to have this catch block here - this is only not supported on the client side.